redaxo/src/core/lib/form/form_base.php

```diff
@@ -41,6 +41,9 @@ abstract class rex_form_base
     /** @var rex_csrf_token */
     private $csrfToken;
 
+    /** @var array<string, string> */
+    private $formAttributes;
```

Comment:
```suggestion
    /** @var array<string, string> */
    private $formAttributes;

    /** @var rex_csrf_token */
    private $csrfToken;
```

So steht es direkt nach `$formId`.